src/background/adblocker.js

```diff
@@ -11,28 +11,22 @@
 
 import { FiltersEngine } from '@cliqz/adblocker';
 import { parse } from 'tldts-experimental';
+import { store } from 'hybrids';
+import Options, { DNR_IDS } from '/store/options.js';
 
-const adblockerEngines = {
-  'ads': {
+const adblockerEngines = DNR_IDS.reduce((map, name) => {
```

Comment:
```js
DNR_IDS.reduce((map, name) => ({
  ...map,
  [map[name]]: {
    engine: null,
    isEnabled: false,
  },
}), {});
```
that said, I still prefer the explicit list more ;-)

Reply:
Here is not about explicitly, we have redundant information, which in fact relays on DNR lists - it turns on the engine only if the DNR list is also turned on, so we should generate the engine list using the DNR ids.